interfaces/builtin/hardware_random_control.go

```diff
@@ -0,0 +1,90 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2017 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"fmt"
+
+	"github.com/snapcore/snapd/interfaces"
+	"github.com/snapcore/snapd/interfaces/apparmor"
+	"github.com/snapcore/snapd/interfaces/udev"
+	"github.com/snapcore/snapd/snap"
+)
+
+const hardwareRandomControlConnectedPlugAppArmor = `
+# Description: allow direct access to the hardware random number generator device. Usually,
+# the default access to /dev/random is sufficient, but this allows applications such as rng-tools
+# to use /dev/hwrng directly or change the hwrng via sysfs. For details, see
+# https://www.kernel.org/doc/Documentation/hw_random.txt
+
+/dev/hwrng rw,
+/sys/devices/virtual/misc/ r,
+/sys/devices/virtual/misc/hw_random/rng_{available,current} r,
+# Allow changing the hwrng
+/sys/devices/virtual/misc/hw_random/rng_current w,
+/run/udev/data/c10:183 r,
```

Comment:
Can you move this to below /dev/hwrng? It corresponds to that device and this access isn't needed for changing the hwrng. I also like to have blcnks lines before comments in policy, for readability. Eg:
```
 +/dev/hwrng rw,
￼ +/run/udev/data/c10:183 r,
￼ +/sys/devices/virtual/misc/ r,
￼ +/sys/devices/virtual/misc/hw_random/rng_{available,current} r,
 +￼
 +# Allow changing the hwrng
￼ +/sys/devices/virtual/misc/hw_random/rng_current w,
```